sqlalchemy_utils/functions/database.py

```diff
@@ -459,41 +462,52 @@ def sqlite_file_exists(database):
 
     url = copy(make_url(url))
     database, url.database = url.database, None
-    engine = sa.create_engine(url)
+    engine = None
+    dialect_name = url.get_dialect().name
 
-    if engine.dialect.name == 'postgresql':
+    if dialect_name == 'postgresql':
+        ret = False
         text = "SELECT 1 FROM pg_database WHERE datname='%s'" % database
-        return bool(get_scalar_result(engine, text))
-
-    elif engine.dialect.name == 'mysql':
+        for pdb in postgres_db:
```

Comment:
```suggestion
        if postgres_db is None:
            postgres_db = ('postgres', 'template0', 'template1')
        for pdb in postgres_db:
```

Reply:
Should we keep `None`? This should use the default, i.e. connect to a db `USERNAME`?

Reply:
Sure, sorry, suggestion updated.